grafana/resource_machine_learning_job_test.go

```diff
@@ -0,0 +1,74 @@
+package grafana
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccResourceMachineLearningJob(t *testing.T) {
+	CheckCloudTestsEnabled(t)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheckCloud(t) },
+		ProviderFactories: testAccProviderFactories,
```

Comment:
For completeness, you can have: 
- a `CheckDestroy` test that checks that the job is actually destroyed after the test: See `testAccFolderCheckDestroy`
- a custom check to test that the job is actually deployed (not just Terraform blindly setting some attributes): `testAccFolderCheckExists`

Reply:
Awesome, thanks for pointing those checks out! I have added a second commit implementing both of them.